indico/core/db/sqlalchemy/locations.py

```diff
@@ -189,3 +189,44 @@ def location_data(self):
             return {'source': data_source, 'room': data_source.room, 'room_name': data_source.room_name,
                     'venue_name': data_source.venue_name, 'address': data_source.address,
                     'inheriting': self.inherit_location}
+
+    @property
+    def location_data_for_UI(self):
```

Comment:
also, don't put this between the getter and setter of the `location_data` property